src/modules/FFmpeg/FFmpeg.cpp

```diff
@@ -233,26 +227,21 @@ ModuleSettingsWidget::ModuleSettingsWidget(Module &module) :
 	allowVDPAUinVAAPIB = new QCheckBox(tr("Allow VDPAU"));
 	allowVDPAUinVAAPIB->setChecked(sets().getBool("AllowVDPAUinVAAPI"));
 
-	QLabel *vaapiDeintMethodL = new QLabel(tr("Improving deinterlacing quality") + ": ");
-
 	vaapiDeintMethodB = new QComboBox;
 	vaapiDeintMethodB->addItems(QStringList() << tr("None") << "Motion adaptive" << "Motion compensated");
 	vaapiDeintMethodB->setCurrentIndex(sets().getInt("VAAPIDeintMethod"));
 	if (vaapiDeintMethodB->currentIndex() < 0)
 		vaapiDeintMethodB->setCurrentIndex(1);
 
-	QGridLayout *vaapiLayout = new QGridLayout(decoderVAAPIEB);
-	vaapiLayout->addWidget(allowVDPAUinVAAPIB, 0, 0, 1, 2);
-	vaapiLayout->addWidget(vaapiDeintMethodL, 1, 0, 1, 1);
-	vaapiLayout->addWidget(vaapiDeintMethodB, 1, 1, 1, 1);
+	QFormLayout *vaapiLayout = new QFormLayout(decoderVAAPIEB);
+	vaapiLayout->addRow(allowVDPAUinVAAPIB);
+	vaapiLayout->addRow(tr("Improving deinterlacing quality") + ": ", vaapiDeintMethodB);
 
 	#ifndef HAVE_VPP
-		vaapiDeintMethodL->setEnabled(false);
 		vaapiDeintMethodB->setEnabled(false);
 	#endif
 #endif
 
-	/* Pospiesz się */
```

Comment:
Don't remove, translate to `/* Hurry up */` (a few lines below there is another `/**/` to close "section")